src/Linker.cs

```diff
@@ -364,9 +364,7 @@ public Function GetDefaultFunction(Store store, string name)
                 return null;
             }
 
-            GC.KeepAlive(store);
-
-            return new Memory(store, ext.of.memory);
+            return store.GetCachedExtern(ext.of.memory);
```

Comment:
I would prefer to keep the `GC.KeepAlive(store)` call here for consistency, as the above code access the `StoreContext`.
Otherwise, when making changes at this location later you would have to reason about whether the `store` is still accessed, or the compiler/GC could optimize it away and finalize the `Store` (like a code debt). What do you think?

Reply:
Added this back in 👍